charts/camunda-platform/test/unit/identity/configmap_test.go

```diff
@@ -124,3 +124,80 @@ func (s *configMapSpringTemplateTest) TestConfigMapExternalDatabaseEnabled() {
 	s.Require().Equal("jdbc:postgresql://my-database-host:2345/my-database-name", configmapApplication.Spring.DataSource.Url)
 	s.Require().Equal("my-database-username", configmapApplication.Spring.DataSource.Username)
 }
+func (s *configMapSpringTemplateTest) TestConfigMapAuthIssuerBackendUrlWhenExplicitlyDefined() {
+	// given
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"global.identity.auth.issuerBackendUrl": "https://fake.com/",
```

Comment:
@jessesimpson36 we also need to explicitly disable Keycloak.
Also, let's avoid using real domains (yeah fack.com is a real one 😄)